packages/wonder-blocks-data/docs.md

```diff
@@ -39,7 +54,7 @@ type CacheEntry =
         data: any,
       |}
     | {|
-        error: string,
+        error: Error,
```

Comment:
Darn it, accidentally undid this fix.

Reply:
OK, fixed it here: https://github.com/Khan/wonder-blocks/commit/8fc15f7964b96cccd35fc36c2a59f4f17dc72863